app/main.py

```diff
@@ -1,2 +1,101 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, hash_val: int, value: Any) -> None:
+        self.key = key
+        self.hash_val = hash_val
+        self.value = value
+        self.next = None
+
+
 class Dictionary:
-    pass
+    def __init__(self, size: int = 8) -> None:
+        self.size = size
+        self.table = [None] * size
+        self.count = 0
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        hash_val = hash(key)
+        index = hash_val % self.size
+        node = self.table[index]
+        while node is not None:
+            if node.key == key:
+                node.value = value
+                return
+            node = node.next
+        new_node = Node(key, hash_val, value)
+        new_node.next = self.table[index]
+        self.table[index] = new_node
+        self.count += 1
+        if self.count > self.size * 2:
+            self._resize()
+
+    def __getitem__(self, key: Any) -> Any:
+        hash_val = hash(key)
+        index = hash_val % self.size
+        node = self.table[index]
+        while node is not None:
+            if node.key == key:
+                return node.value
+            node = node.next
+        raise KeyError(key)
+
+    def __len__(self) -> int:
+        return self.count
+
+    def __contains__(self, key: Any) -> bool:
+        return True if self.get(key) else False
```

Comment:
```suggestion
        return bool(self.get(key))
```

Reply:
Thanks for the improvement issue , I adjusted it in my code. I committed the changes, and again pushed them to GitHub.